make/photon/prepare/templates/jobservice/config.yml.jinja

```diff
@@ -20,6 +20,7 @@ worker_pool:
     #redis://[arbitrary_username:password@]ipaddress:port/database_index
     redis_url: {{redis_url}}
     namespace: "harbor_job_service_namespace"
+    idle_timeout_second: 3600
```

Comment:
I suggest this property can be hard coded and we don't need to expose it in the config YAML file

Reply:
What's the considerations ? I don't think there will be a fixed value for it, it's related to the proxy timeout settings. Won't it be more flexible to give users a chance to configure it ?

Reply:
I'm ok as long as it's not configurable in `harbor.yml`

Reply:
I don't understand as to why someone would want to hardcode this?
Default is 30s in most haproxy charts. So why hardcode 3600?
It would make a ton of sense to give user the chance to change this